services/libs/data-access-layer/src/members/segments.ts

```diff
@@ -66,20 +66,20 @@ export async function fetchManyMemberSegments(
   )
 }
 
-export async function fetchMemberAggregates(
```

Comment:
I use this one in my branch, don't remove it

Reply:
Or maybe we can convert it into something more generic like that it aggregates other fields?

Reply:
I need it to get absolute aggregates too, but including `lastActive`